code/modules/admin/admin_mass_del_in_zone.dm

```diff
@@ -0,0 +1,37 @@
+// Delete all atoms of a given type in a zone
+
+/datum/admins
+	var/x_min_del = 0
+	var/x_max_del = 0
+	var/y_min_del = 0
+	var/y_max_del = 0
+	var/z_del = 0
+	var/type_del = null
+
+/datum/admins/proc/mass_delete_in_zone()
+	set category = "Server"
+	set desc="Delete all atoms of a given type in a zone."
+	set name="Mass delete in zone"
+	if (!check_rights(R_SERVER))
+		to_chat(src, "<span class='warning'>You need +SERVER to do this.</span>")
+		return FALSE
+
+	src = usr.client.holder // why lummox why
+	
+	var/list/dat = list()
+	dat += "<h3>Mass deletion in a zone</h3>"
```

Comment:
This can actually be made into a single var, hang on

Reply:
Just use document syntax